### ISSUE TITLE
Start Countdown Bug

### ISSUE THREAD

**Issue by TheMan6900** [EmeraldLoc/tag#888]
![image](https://github.com/user-attachments/assets/160130d5-a606-4f7a-8f9a-33dbb00b0151)

**Comment by EmeraldLoc** [comment]
Ye this is known, just need to send a packet when a mod starts a game.

**Comment by TheMan6900** [comment]
these ig
![image](https://github.com/user-attachments/assets/6e6eb64c-59f6-4500-88e6-67a124ba0ac6)

**Comment by TheMan6900** [comment]
we had auto mode off, btw

**Comment by EmeraldLoc** [comment]
No, there aren’t any packets being sent rn, need to add that. Although that in of itself is a bug.

**Comment by TheMan6900** [comment]
> we had auto mode off, btw

this can happen with auto mode on, btw

**Comment by TheMan6900** [comment]
do you know anything new about this bug?

**Comment by TheMan6900** [comment]
Emerald please fix this.........
![image](https://github.com/user-attachments/assets/7b665988-f60a-4a7b-8692-0a14acb8924a)

**Comment by TheMan6900** [comment]
The timer can also be lower than intented

**Comment by EmeraldLoc** [comment]
Yes I know. I will fix it when I do. Dont beg for things to be resolved that'll just make me more frustrated.

**Comment by TheMan6900** [comment]
> Emerald please fix this......... ![image](https://private-user-images.githubusercontent.com/168046906/369638842-7b665988-f60a-4a7b-8692-0a14acb8924a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY5NTI1NDAsIm5iZiI6MTcyNjk1MjI0MCwicGF0aCI6Ii8xNjgwNDY5MDYvMzY5NjM4ODQyLTdiNjY1OTg4LWY2MGEtNGE3Yi04NjkyLTBhMTRhY2I4OTI0YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyMVQyMDU3MjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wNTc0ZTNmZDExNmZiN2Y3MWQ3MDFmZmU5OGJmNDI2NTBhODIyN2EzMGM1NWIxY2ZlY2YxN2YzNTNkNDc1YWVkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.LSkdrbjZs4XVPqotJ-s8hl6fQ9OhMgrGY63jS0V-zmo)

I didn't actually mean this that seriously, I was more just using it as a segway into talking about something else I found out about the problem.

**Comment by TheMan6900** [comment]
Pretty sure what happens when the timer becomes lower than normal, is from the timer not resetting to full upon round start
